juju/client/proxy/kubernetes/proxy.py

```diff
@@ -67,13 +65,16 @@ def connect(self):
 
     def __del__(self):
         self.close()
+        try:
+            if self.temp_ca_file:
+                os.unlink(self.temp_ca_file)
+        except FileNotFoundError:
```

Comment:
I don't think we need this exception handling. Python already ignores (and prints to stderr) exceptions that are raised in `__del__`, for example:

```python
Exception ignored in: <function C.__del__ at 0x72cfa9b4b6a0>
Traceback (most recent call last):
  File "<stdin>", line 4, in __del__
RuntimeError: problem!
```

Reply:
done.